pom.xml

```diff
@@ -28,6 +28,7 @@
         <groovyAllVersion>2.4.7</groovyAllVersion>
         <lombokVersion>1.18.12</lombokVersion>
         <spockVersion>1.1-groovy-2.4</spockVersion>
+        <version.number>1.${git.total.commit.count}.0_${git.commit.id.abbrev}</version.number>
```

Comment:
do we really want to use this version pattern, it is rather uncommon i think, what do you think of semantic versioning for this case?

Reply:
Yes, we will support semantic versioning at the end.